src/control/settings/Settings.cpp

```diff
@@ -596,6 +598,15 @@ auto Settings::load() -> bool {
     loadButtonConfig();
     loadDeviceClasses();
 
+    paletteFilePath = PALETTE_FILE;
+    auto paletteFile = Util::getConfigFile(paletteFilePath);
```

Comment:
```suggestion
    auto paletteFile = Util::getConfigFile(PALETTE_FILE);
```

Reply:
I did this to give easy access to `paletteFilePath`  for every code having access to Settings. But as this is nowhere used right now I'm open to removing it. It would only be helpful for future functionality of showing palette info, or even selecting it dynamically. What do you think?

Reply:
It's unlikely that it is ever needed. And if we will decide to store such strings/paths in the future, they should be stored at another position. 
Also, it is very cheap to recreate the path.

Reply:
Ok. You convinced me. Will be done :heavy_check_mark: